packages/react-provider/src/stories/FluentProviderDirNested.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { makeStyles } from '@fluentui/react-make-styles';
+import { FluentProvider } from '@fluentui/react-provider'; // codesandbox-dependency: @fluentui/react-provider ^9.0.0-beta
+
+const useStyles = makeStyles({
+  example: {
+    margin: '5px',
+  },
+  darkThemeBackground: theme => ({
+    backgroundColor: theme.colorBrandBackground,
+  }),
+});
+
+export const DirNested = () => {
+  const styles = useStyles();
+  return (
+    <>
+      <FluentProvider dir="rtl">
+        <div className={styles.example}>نص من اليمين إلى اليسار</div>
+        <FluentProvider dir="ltr">
+          <div className={styles.example}>Text left to right</div>
+        </FluentProvider>
+      </FluentProvider>
+    </>
+  );
+};
+
+DirNested.parameters = {
+  docs: {
+    description: {
+      story: 'Nested Fluent providers can have different LTR and RTL layout.',
```

Comment:
I am not sure that we should have an example for this behavior at all. I suggest to remove it for now.